<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>imageUIClass</class>
 <widget class="QMainWindow" name="imageUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1073</width>
    <height>642</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Manipulator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>561</width>
      <height>591</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="picture">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>17</y>
      <width>481</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="readInImage">
         <property name="text">
          <string>Read In Image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="imageName">
         <property name="text">
          <string>Enter .pgm file</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="enlargeImage">
         <property name="text">
          <string>Enlarge </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="enlarge_LineEdit"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="2" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="shrinkImage">
         <property name="text">
          <string>Shrink </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="shrink_LineEdit"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="3" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="translateImage">
         <property name="text">
          <string>Translate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="translate_LineEdit"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="4" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="rotateImage">
         <property name="text">
          <string>Rotate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="row_LineEdit">
         <property name="text">
          <string>Enter row</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="col_LineEdit">
         <property name="text">
          <string>Enter column</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="rotation_LineEdit">
         <property name="text">
          <string>Enter theta</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QPushButton" name="reflectImage">
           <property name="text">
            <string>Reflect</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="reflectEdit">
         <property name="text">
          <string>Reflect H or V</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="1">
      <spacer name="horizontalSpacer_10">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>208</width>
         <height>24</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="6" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QPushButton" name="meanGray">
         <property name="text">
          <string>Mean Gray</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="meanGray_label">
         <property name="text">
          <string>--------</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_9">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="7" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QPushButton" name="negateImage">
         <property name="text">
          <string>Negate</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="8" column="0" colspan="2">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QPushButton" name="subImage">
         <property name="text">
          <string>Sub Image</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="UL_Label">
         <property name="text">
          <string>Upper Col:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLineEdit" name="UL_LineEdit"/>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="UR">
         <property name="text">
          <string>Upper Row:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLineEdit" name="UR_LineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="LL_Label">
         <property name="text">
          <string>Lower Col:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLineEdit" name="LL_LineEdit"/>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="LR">
         <property name="text">
          <string>Lower Row:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QLineEdit" name="LR_LineEdit"/>
       </item>
      </layout>
     </item>
     <item row="9" column="0" colspan="2">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="3">
        <widget class="QLabel" name="Regions_label">
         <property name="text">
          <string>Count/Label Regions: </string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="thresholdValue_LineEdit">
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="label_num">
         <property name="text">
          <string>Number of objects:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QLabel" name="threshold_label">
         <property name="text">
          <string>--------</string>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>13</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QPushButton" name="BFS">
         <property name="text">
          <string>Iterative BFS</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="DFS">
         <property name="text">
          <string>Iterative DFS</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4" colspan="2">
        <widget class="QPushButton" name="Recursive">
         <property name="text">
          <string>Recursive</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="Elapsed_label">
         <property name="text">
          <string>Time Elapsed:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="time_label">
         <property name="text">
          <string>--------</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="5">
        <spacer name="horizontalSpacer_11">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>308</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="10" column="0">
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="mergeImage">
         <property name="text">
          <string>Merge With...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="imageName_2">
         <property name="text">
          <string>Enter .pgm file</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="11" column="0" colspan="2">
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="differenceImage">
         <property name="text">
          <string>Difference With...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="imageName_3">
         <property name="text">
          <string>Enter .pgm file</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="12" column="0">
      <layout class="QFormLayout" name="formLayout_3">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QPushButton" name="saveAsImage">
         <property name="text">
          <string>Save As...</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="imageName_4">
         <property name="text">
          <string>Enter .pgm file</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="13" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QPushButton" name="quitImage">
         <property name="text">
          <string>Quit</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1073</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
